### ISSUE TITLE
Add warning for dangerous delete of gcode blocks

### ISSUE THREAD

**Issue by lhartmann** [vlachoudis/bCNC#777]
While not properly a bug, deleting gcode blocks may take away M3, F or other important commands and crash the machine.

Request: Issue a warning when a deleted gcode block contains anything other than simple move commands.

**Comment by vlachoudis** [comment]
Good point

**Comment by masterleo** [comment]
Good point ! I crashed my CNC 3 times because of moving a block that contains M3.
